packages/protocol/contracts/governance/Election.sol

```diff
@@ -917,7 +875,7 @@ contract Election is
    * @notice Returns get current validator signers using the precompiles.
    * @return List of current validator signers.
    */
-  function getCurrentValidatorSigners() public view onlyL1 returns (address[] memory) {
+  function getCurrentValidatorSigners() public view returns (address[] memory) {
```

Comment:
Is it a deliberate decision to keep this function? It was slated as `onlyL1` before, though I guess it can still function given the precompile overrides.  The same functionality can be found in EpochManager's `getElectedAccounts` (and so if we *do* want to keep this function in this contract, might be more efficient to call into that?).